client/state/data-layer/wpcom/read/tags/mine/delete/index.js

```diff
@@ -31,34 +29,40 @@ export function requestUnfollow( store, action ) {
  * @param  {RemovedTag} apiResponse api response from the unfollow
  * @return {Number} the ID of the tag that was removed
  */
-export const fromApi = apiResponse => apiResponse.removed_tag;
-
-export function receiveUnfollowTag( store, action, apiResponse ) {
+export const fromApi = apiResponse => {
 	if ( apiResponse.subscribed ) {
-		receiveError( store, action );
-		return;
+		throw new Error(
+			`failed to unsubscribe to tag with reponse: ${ JSON.stringify( apiResponse ) }`
```

Comment:
s/reponse/response

Reply:
saving bits.

kidding! Thanks I'll fix that

Reply:
:D